setup.py

```diff
@@ -10,7 +10,7 @@
     long_description = readme_file.read()
 
 
-VERSION = "13.7.4"
+VERSION = "13.8.0"
```

Comment:
No need to change this. The tooling will

Reply:
thanks @bdraco should I remove it?